tdms/src/iterator.cpp

```diff
@@ -5971,56 +5793,54 @@ void mexFunction(int nlhs, mxArray *plhs[], int nrhs, const mxArray *prhs[]) {
   *mxGetPr((mxArray *) plhs[25]) = maxfield;
 
   if (runmode == rm_complete && exphasorsvolume) {
-    setGridLabels(input_grid_labels, output_grid_labels, pind_il, pind_iu, pind_jl, pind_ju,
-                  pind_kl, pind_ku);
+    setGridLabels(input_grid_labels, output_grid_labels, E.il, E.iu, E.jl, E.ju, E.kl, E.ku);
   }
-
-
+  
   auto interp_output_grid_labels = GridLabels();
 
   //fprintf(stderr,"Pos 15_m1\n");
   if (runmode == rm_complete && exphasorsvolume) {
     //now interpolate over the extracted phasors
     if (dimension == THREE) {
-      fprintf(stderr, "mxInterpolateFieldCentralE: %d %d %d \n", pind_iu - pind_il - 1,
-              pind_ju - pind_jl - 1, pind_ku - pind_kl - 1);
+      fprintf(stderr, "mxInterpolateFieldCentralE: %d %d %d \n", E.I_tot - 2,

```

Comment:
Shouldn't these be `E.iu`, `E.il` etc. instead of `tot`?

Reply:
And similar for all the changed lines below in this block

Reply:
I think this is okay:

```c++
 E.I_tot = E.iu - E.il + 1 = pind_iu - pind_il + 1
```
so
```c++
 E.I_tot -2 == pind_iu - pind_il -1
```